packages/dataviews/src/components/dataform/stories/index.story.tsx

```diff
@@ -26,6 +28,23 @@ import { unlock } from '../../../lock-unlock';
 
 const { ValidatedTextControl } = unlock( privateApis );
 
+
+const DollarPrefix = () => (
+	<InputControlPrefixWrapper>
+		<span>$</span>
+	</InputControlPrefixWrapper>
+);
+const PercentSuffix = () => (
```

Comment:
Can we make one of these use an icon to demonstrate the usage of the icon variant?